gnomad_qc/example_notebooks/ancestry_classification_using_gnomad_rf.ipynb

```diff
@@ -0,0 +1,784 @@
+{
```

Comment:
<p>Maybe add an intro description like this?</p><blockquote>For the v3 example, we will try to assign the genetic ancestry for a test dataset of gnomAD v4 based on v3.1 pre-computed PCs and v3.1 ONNX random classifier model. We will start with the test dataset in raw VDS format: select necessary entry annotations, split multiallelic variants, filter the overlapping variants with the PCA loadings HT of the reference dataset, densify the VDS to a MT, filter the MT on 'adj' thresholds of GQ/DP/AB; then, we will project the genotypes onto pre-computed v3.1 PCs to get the PCA scores; lastly, we will assign the genetic ancestry with the v3.1 ONNX random forest classifier model.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/broadinstitute/gnomad_qc/pull/366/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>